[2024-04-07T03:55:55.542+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: setup.set_up_datawarehouse manual__2024-04-07T03:54:58.082620+00:00 [queued]>
[2024-04-07T03:55:55.546+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: setup.set_up_datawarehouse manual__2024-04-07T03:54:58.082620+00:00 [queued]>
[2024-04-07T03:55:55.546+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-04-07T03:55:55.551+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): set_up_datawarehouse> on 2024-04-07 03:54:58.082620+00:00
[2024-04-07T03:55:55.555+0000] {standard_task_runner.py:60} INFO - Started process 1239 to run task
[2024-04-07T03:55:55.557+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'setup', 'set_up_datawarehouse', 'manual__2024-04-07T03:54:58.082620+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/setup.py', '--cfg-path', '/tmp/tmpjgdu3x0x']
[2024-04-07T03:55:55.559+0000] {standard_task_runner.py:88} INFO - Job 77: Subtask set_up_datawarehouse
[2024-04-07T03:55:55.581+0000] {task_command.py:423} INFO - Running <TaskInstance: setup.set_up_datawarehouse manual__2024-04-07T03:54:58.082620+00:00 [running]> on host befaceac7652
[2024-04-07T03:55:55.615+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='setup' AIRFLOW_CTX_TASK_ID='set_up_datawarehouse' AIRFLOW_CTX_EXECUTION_DATE='2024-04-07T03:54:58.082620+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-07T03:54:58.082620+00:00'
[2024-04-07T03:55:55.670+0000] {warehouse.py:38} ERROR - Error while creating/dropping databases!
[2024-04-07T03:55:55.707+0000] {warehouse.py:75} ERROR - 1046 (3D000): No database selected
[2024-04-07T03:55:55.740+0000] {warehouse.py:75} ERROR - 1046 (3D000): No database selected
[2024-04-07T03:55:55.769+0000] {warehouse.py:38} ERROR - Error while creating/dropping databases!
[2024-04-07T03:55:55.804+0000] {warehouse.py:75} ERROR - 1046 (3D000): No database selected
[2024-04-07T03:55:55.835+0000] {warehouse.py:75} ERROR - 1046 (3D000): No database selected
[2024-04-07T03:55:55.873+0000] {warehouse.py:38} ERROR - Error while creating/dropping databases!
[2024-04-07T03:55:55.906+0000] {warehouse.py:75} ERROR - 1046 (3D000): No database selected
[2024-04-07T03:55:55.942+0000] {warehouse.py:75} ERROR - 1046 (3D000): No database selected
[2024-04-07T03:55:55.942+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-07T03:55:55.947+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=setup, task_id=set_up_datawarehouse, execution_date=20240407T035458, start_date=20240407T035555, end_date=20240407T035555
[2024-04-07T03:55:55.983+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-07T03:55:55.996+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
